test/scripts/base-host-check.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -euxo pipefail
```

Comment:
I think at some point we used `systemctl is-system-running --wait` as a sanity check, might that be appropriate here? It would catch any services that failed to start.

Reply:
Added.